apps/E2E/src/ButtonExperimental/pages/ButtonExperimentalPageObject.ts

```diff
@@ -34,10 +36,20 @@ class ButtonExperimentalPageObject extends BasePage {
 
   /* Returns the correct WebDriverIO element from the Button Selector */
   async getButtonSelector(buttonSelector?: ButtonSelector): Promise<WebdriverIO.Element> {
-    if (buttonSelector == ButtonSelector.PrimaryButton) {
-      return await this._primaryComponent;
+    switch (buttonSelector) {
+      case ButtonSelector.PrimaryButton:
+        return this._primaryComponent;
+      case ButtonSelector.SecondaryButton:
+        return this._secondaryComponent;
     }
-    return await this._primaryComponent;
+  }
+
+  async accessibilityLabelIsEqualTo(selector: ButtonSelector, expectedValue: any): Promise<boolean> {
```

Comment:
These two functions will be re-used for every single spec. How can we make this re-usable without creating new definitions for each page object?

Reply:
I'm thinking we should move this function to BasePage, and change the first parameter to `element: WebDriverIO.Element`. And then from the spec, we can call:

`ButtonExperimentalPageObject.accessibilityLabelIsEqualTo(ButtonExperimentalPageObject._primaryComponent, BUTTON_ACCESSIBILITY_LABEL)`

or

`ButtonExperimentalPageObject.accessibilityLabelIsEqualTo(ButtonExperimentalPageObject.getButtonSelector(ButtonSelector.PrimaryButton), BUTTON_ACCESSIBILITY_LABEL)`

Reply:
We could probably directly pass in a WebdriverIO element and ditch the selector.

Is there a reason we need to use a Selector enum for things like `sendKey()` or `clickComponent()`?

Reply:
No, we could change those too. All these functions should probably take in a WebDriverIO.Element rather than a page-specific selector